<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="360LP048" name="VerifyNameEmailInUserTableIfUserRegisteredOurWebsite" tests="2" failures="1" timestamp="21 Oct 2016 10:21:29 GMT" time="395.126" errors="0">
  <testcase name="test_004VerifyCampaignAndLevelAssignment" time="81.849" classname="com.edureka.referralProcess.VerifyNameEmailInUserTableIfUserRegisteredOurWebsite"/>
  <testcase name="test_017VerifyNameEmailInUserTableIfUserRegisteredOurWebsite" time="297.163" classname="com.edureka.referralProcess.VerifyNameEmailInUserTableIfUserRegisteredOurWebsite">
    <failure type="com.mysql.jdbc.exceptions.jdbc4.CommunicationsException" message="Communications link failure

Last packet sent to the server was 0 ms ago.">
      <![CDATA[com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

Last packet sent to the server was 0 ms ago.
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1074)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2104)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:729)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:283)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.edureka.util.Database.getDatabaseConnection(Database.java:29)
	at com.edureka.util.Database.<init>(Database.java:22)
	at com.edureka.pages.DatabaseVerifications.checkUserIsRegistered(DatabaseVerifications.java:942)
	at com.edureka.pages.MyProfilePage.verifyUserRegisteredOrNot(MyProfilePage.java:643)
	at com.edureka.referralProcess.VerifyNameEmailInUserTableIfUserRegisteredOurWebsite.test_017VerifyNameEmailInUserTableIfUserRegisteredOurWebsite(VerifyNameEmailInUserTableIfUserRegisteredOurWebsite.java:361)
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at java.net.Socket.connect(Socket.java:538)
	at java.net.Socket.<init>(Socket.java:434)
	at java.net.Socket.<init>(Socket.java:244)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:256)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:276)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2027)
	... 39 more
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test_017VerifyNameEmailInUserTableIfUserRegisteredOurWebsite -->
</testsuite> <!-- VerifyNameEmailInUserTableIfUserRegisteredOurWebsite -->
